docs/api.md

```diff
@@ -135,6 +135,7 @@ With widgets, Streamlit allows you to bake interactivity directly into your apps
 .. autofunction:: streamlit.text_area
 .. autofunction:: streamlit.date_input
 .. autofunction:: streamlit.time_input
+.. autofunction:: streamlit.sidebar
```

Comment:
I believe this is not going to work as you expect. The sidebar is just a DeltaGenerator object, so the docs here will talk about general DeltaGenerator stuff instead of sidebar-specific things. 

If you regenerate the docs locally and think this reads well, then feel free to keep it. My guess is it's not going to work out though...

Reply:
Yea, realized that after our email thread last week. Making adjustments to the file now. Should be fixed in the next push.